### ISSUE TITLE
Grading

### ISSUE THREAD

**Issue by bhantos** [patbaa/physdl#1]
Is it necessary to turn in both assignments (and reach the minimum of the 1 point baseline on both of those assignments) for getting a grade?

**Comment by patbaa** [comment]
Yes, please submit for both and try to beat the 1 point baseline at least. Especially, as this course is mainly focusing on deep learning which is the topic of the second challenge only.
